File: common/retex/src/main/kotlin/com/himamis/retex/renderer/android/parser/AttrA.java
package com.himamis.retex.renderer.android.parser;

import com.himamis.retex.renderer.share.platform.parser.Attr;

import org.w3c.dom.Node;

public class AttrA extends NodeA implements Attr {

    public org.w3c.dom.Attr impl;

    public AttrA(org.w3c.dom.Attr impl) {
        super((Node) impl);
        this.impl = impl;
    }

    public String getName() {
        return impl.getName();
    }

    public boolean isSpecified() {
        return impl.getSpecified();
    }

    public String getValue() {
        return impl.getValue();
    }

}

File: modules/retex/src/main/kotlin/com/himamis/retex/renderer/android/parser/AttrA.kt
package com.himamis.retex.renderer.android.parser

import org.w3c.dom.Attr
import org.w3c.dom.Node

class AttrA(var impl: Attr) : NodeA(impl as Node),
    com.himamis.retex.renderer.share.platform.parser.Attr {
    override fun getName(): String {
        return impl.name
    }

    override fun isSpecified(): Boolean {
        return impl.specified
    }

    override fun getValue(): String {
        return impl.value
    }
}
